jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -63,7 +63,7 @@
           new IllegalStateException("invalid usage; required step not configured"));
     }
 
-    private Future<ImageAndRegistryClient> baseImageAndRegistryClient = failedFuture();
+    private Future<ImagesAndRegistryClient> imagesAndRegistryClient = failedFuture();
```

Comment:
So in this case, we should name it `baseImages...` as an instance that belongs to the `ImagesAndRegistryClient` "class". This instance will all contain base images.